src/utils/ConfigUtils.js

```diff
@@ -147,6 +147,49 @@ const isInstanceViewConfigValid = (instanceView) => {
   return true;
 };
 
+const checkUnauthorizedKeysInConfig = (schema, data) => {
+  try {
+    schema.parse(data);
+  } catch (e) {
+    e.issues
+      .filter((issue) => issue.code === 'unrecognized_keys')
+      .forEach((issue) => {
+        if (issue.path?.length === 0) {
+          console.warn(
+            `Your solution contains one or more unrecognized keys: '${issue.keys.join(
+              ', '
+            )}', please, check your configuration`
+          );
+        } else {
+          if (issue.path?.includes('parameters')) {
+            const parameterId = data.parameters[issue.path[1]].id;
```

Comment:
Normally if `data` is `null`, there is no validation errors because `null` is acceptable as type, errors will be displayed in case there are some keys in `data`. But you are right, it's better to add an `? `just in case :+1: :slightly_smiling_face: